tests/foreman/ui/test_registration.py

```diff
@@ -42,6 +49,7 @@ def test_positive_verify_default_values_for_global_registration(
     assert cmd['general']['location'] == 'Default Location'
     assert cmd['general']['capsule'] == 'Nothing to select.'
     assert cmd['general']['operating_system'] == default_os.title
+    assert cmd['general']['operating_system'] == ''
```

Comment:
Why the double assertion?

Reply:
There was a cherry-pick issue. Corrected it.